server/db/user_models/user.js

```diff
@@ -14,40 +14,84 @@ const Users = db.define('users', {
     primaryKey: true
   },
   firstName: {
-    type: STRING
+    type: STRING,
+    allowNull: false,
```

Comment:
why wasnt this here from the beginnning?

Reply:
we removed some validation to make manual testing faster (I'm ready to be yelled at for this :) )